front/app/containers/ProjectFolderShowPage/index.tsx

```diff
@@ -120,108 +121,101 @@ const CardsWrapper = styled.div`
   background: ${colors.background};
 `;
 
-const ProjectFolderShowPage = memo<{
-  projectFolder: IProjectFolderData | null | undefined;
-}>(({ projectFolder }) => {
-  const { data: authUser } = useAuthUser();
-  const { windowWidth } = useWindowSize();
-  const smallerThan1280px = windowWidth ? windowWidth <= 1280 : false;
+interface Props {
+  projectFolder: IProjectFolderData;
+}
 
-  const loading = projectFolder === undefined;
+const ProjectFolderShowPage = ({ projectFolder }: Props) => {
+  const { data: authUser } = useAuthUser();
+  const isSmallerThanSmallDesktop = useBreakpoint('smallDesktop');
```

Comment:
Standardizing breakpoints. Tested the page with the new breakpoint. Basically, we're going to the mobile version a bit sooner.